science/libsbml/Portfile

```diff
@@ -18,32 +19,117 @@ long_description    LibSBML is a free, open-source programming library to \
                     rather a library you can embed in your own applications.
 
 homepage            http://sbml.org/Software/libSBML
-master_sites        sourceforge:project/sbml/libsbml/${version}/stable
+master_sites        sourceforge:project/sbml/libsbml/${version}/experimental/source
```

Comment:
Why did you change the master_sites to point to the experimental directory? 5.16.0 is the latest stable version and is in the stable directory like original master_sites. Please restore the master_sites line to the original value.

Reply:
Thank you for the comment.

As you can see in https://sourceforge.net/projects/sbml/files/libsbml/5.16.0/, the difference between `experimental` and `stable` is whether it includes the code of beta packages (SBML extensions) or not (which means, there is no difference in SBML-core).

The reason why I have used the experimental code is that I wanted to support SBML extension packages (arrays, distrib, dyn, render, req, and spatial) in this port. The extension packages are not installed by default unless the user adds some variants (+arrays +distrib +dyn etc.).

Moreover, the code between experimental and stable is identical except the above extension packages. This means that there is no difference in generated libSBML if no variants on the extension is added. Here are the md5 value of `libsbml.5.16.0.dylib` on both stable and experimental. Both libsbml.5.16.0.dylib was generated with the same cmake option (without any variants).

```
% md5 stable/build/src/libsbml.5.16.0.dylib experimental/build/src/libsbml.5.16.0.dylib
MD5 (stable/build/src/libsbml.5.16.0.dylib) = d67fcbf2229c61d9b12d0cb875554717
MD5 (experimental/build/src/libsbml.5.16.0.dylib) = d67fcbf2229c61d9b12d0cb875554717
```

Here is the command line I used to check the difference between stable and experimental, and the diff file is also attached to this message.

```
% diff -urBb libSBML-5.16.0-Source-experimental/src libSBML-5.16.0-Source-stable/src > src.diff
```

Please let me know if you have a further question about it.

[src.diff.gz](https://github.com/macports/macports-ports/files/1737743/src.diff.gz)

Reply:
@mf2k Opinions? Should we pull at this point?

Reply:
Yes, I'm good with this.